docs/components/dartpad/tooltips/regular/dartpad_metadata.yaml

```diff
@@ -0,0 +1,4 @@
+name: Material Tooltips Demonstration
+mode: flutter
+files:
+  - name: main.dart
```

Comment:
new line

Reply:
where should there be a new line? between 3 and 4?

Reply:
Below. There should always be a new empty line at the end of files